ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__amd64-nightly.yaml

```diff
@@ -2621,6 +2621,11 @@ tests:
   steps:
     cluster_profile: gcp-qe
     env:
+      COMPUTE_NODE_TYPE: c3-standard-4
```

Comment:
Maybe we also need to update z-stream upgrade ci jobs for the same install workflow to keep consistent.

Reply:
Updated `periodic-ci-openshift-openshift-tests-private-release-4.19-amd64-nightly-4.19-upgrade-from-stable-4.19-gcp-ipi-secureboot-confidential-computing-mini-perm-f60` accordingly, thanks!